src/beam3/4C_beam3_euler_bernoulli_evaluate.cpp

```diff
@@ -271,7 +271,8 @@ int Discret::Elements::Beam3eb::evaluate_neumann(Teuchos::ParameterList& params,
     time = params.get("total time", -1.0);
 
   // find out whether we will use a time curve and get the factor
-  const auto* tmp_funct = &condition.parameters().get<std::vector<int>>("FUNCT");
+  const auto* tmp_funct =
+      &condition.parameters().get<std::vector<Core::IO::Noneable<int>>>("FUNCT");
```

Comment:
Since you reworked this anyway, would you mind removing the pointless pointers here? I leave it to you.